apps/showcase/project.json

```diff
@@ -55,7 +55,7 @@
         "{projectRoot}/dev-resources",
         "{projectRoot}/training-assets",
         "{projectRoot}/*.metadata.json",
-        "packages/@o3r/training-sdk/dist/*-structure.json"
+        "{workspaceRoot}/packages/@o3r/training-sdk/dist/*-structure.json"
```

Comment:
Not sure it is a good idea to add a file from another page as input.
The best practice is to add the task, from `@o3r/training-sdk`, resulting to these json files in `dependOn` with the `^` prefix.

Reply:
Updated (this will be more clear when we open the PR for with the package @o3r/training-sdk --> I can remove this modification in the current PR and add it to the next one if you prefer)